### PR TITLE
Merge dev into master

### PR DESCRIPTION
**Pull request by bleaker0x**
Implements token swaps via the SDK, as well as fixing #39.

### CODE REVIEW

**Comment by BlazeWasHere** [comment]
Waiting on #46 to be merged first before merging this into master.

**Review by BlazeWasHere — APPROVED** [review]

**Comment by bleaker0x** [comment]
@BlazeWasHere Not sure #46 is actually going to be merged; see my comment on it there.

**Pull request merged**
